doc(upgrade): dismiss button known issue (#736)
harvester/docs: docs/upgrade/v1-3-2-to-v1-4-0.md

```shell
helm rollback fleet -n cattle-fleet-system <last-deployed-revision>
```
---

> ### 4. Upgrade will start over again unexpectedly after clicking the "Dismiss it" button
> 
> When performing Harvester upgrades on the external Rancher dashboard, clicking the "Dismiss it" button in the upgrade dialog of the Rancher dashboard UI may trigger an unexpected restart of the upgrade process.
> 
> To avoid this issue, use the standalone Harvester dashboard user interface instead. Selecting the "Dismiss it" button within this interface will dismiss the dialog correctly and resolve the matter without complications.
> 
> Alternatively, instead of using the "Dismiss it" button, you can achieve the same result by executing the following command against the cluster:

Reviewer: ```suggestion
### 4. Upgrade unexpectedly restarts after Rancher UI button is clicked

When you use Rancher to upgrade Harvester, the Rancher UI displays a dialog with a button labeled "Dismiss it". Clicking this button may trigger an unexpected restart of the upgrade process.

To avoid this issue, perform either of the following actions:

- Use the Harvester UI to upgrade Harvester. Clicking the "Dismiss it" button on the Harvester UI does not result in unexpected behavior.
- Instead of clicking the button on the Rancher UI, run the following command against the cluster:
```

```shell
helm rollback fleet -n cattle-fleet-system <last-deployed-revision>
```
---

### 4. Upgrade will start over again unexpectedly after clicking the "Dismiss it" button

> When you use Rancher to upgrade Harvester, the Rancher UI displays a dialog with a button labeled "Dismiss it". Clicking this button may trigger an unexpected restart of the upgrade process.

Reviewer 2: >Clicking this button may trigger an unexpected restart of the upgrade process.

Clicking this button will wipe the `upgrade.harvesterhci.io` object's status, important information like below is lost.

```
  status:
   ...
    previousVersion: v1.5-cb719430-head
    repoInfo: |
      release:
        harvester: cb719430
        harvesterChart: 0.0.0-v1.5-cb719430
        os: Harvester v1.5
        kubernetes: v1.32.2+rke2r1
        rancher: v2.11.0-rc5
        monitoringChart: 105.1.2+up61.3.2
```
The another side-effect is: it may trigger the unexpected restart of the upgrade process when `upgradeLog` is not enabled. When `upgradeLog` is enabled, it will not.

```shell
helm rollback fleet -n cattle-fleet-system <last-deployed-revision>
```
---

### 4. Upgrade will start over again unexpectedly after clicking the "Dismiss it" button

When you use Rancher to upgrade Harvester, the Rancher UI displays a dialog with a button labeled "Dismiss it". Clicking this button may result in the following issues:

1. The `harvesterhci.io/v1beta1/upgrade` CR's `status` section will be wiped out, thus losing all the important information about the upgrade process.
2. The upgrade process will start over again unexpectedly.

> You will be affected by this issue if the Rancher version is v2.10.x (the incorporated Harvester UI Extension version is v1.0.2, v1.0.3, and v1.0.4). For more details, see [Harvester UI Extension Support Matrix](../rancher/harvester-ui-extension.md#support-matrix). The issue will be fixed starting from Harvester UI Extension version v1.0.5 and v1.5.0. All Harvester standalone UI versions are not affected.

Reviewer: ```suggestion
This issue affects Rancher v2.10.x, which uses v1.0.2, v1.0.3, and v1.0.4 of the [Harvester UI Extension](../rancher/harvester-ui-extension.md#support-matrix). All Harvester UI versions are not affected. The issue will be fixed in Harvester UI Extension v1.0.5 and v1.5.0.
```
